app/code/Magento/Config/Block/System/Config/Form.php

```diff
@@ -797,6 +800,24 @@ private function getAppConfig()
         return $this->appConfig;
     }
 
+    /**
+     * Check Default Path Readonly
+     *
+     * @param string $path
+     * @return boolean
+     */
+    private function isDefaultFieldReadOnly($path)
+    {
+        $scope = $this->getScope();
+        $isReadOnly = false;
+        if ($scope !== ScopeConfigInterface::SCOPE_TYPE_DEFAULT) {
```

Comment:
I think this check is unnecessary, `$isReadOnly` value assigned inside the condition is correct for default scope

Reply:
Removed this check.